go/src/koding/db/mongodb/modelhelper/credentials.go

```diff
@@ -61,6 +83,12 @@ func UpdateCredential(identifier string, data bson.M) error {
 	})
 }
 
+func CreateCredential(cred *models.Credential) error {
+	return Mongo.Run(CredentialsColl, func(c *mgo.Collection) error {
+		return c.Insert(cred)
```

Comment:
Currently no Create\* function does that, they assume the caller is responsible for creating / passing valid models.\* struct. Could we leave that as-is to be consistent?